### ISSUE TITLE
Add is_test to TickerDetails model

### ISSUE THREAD

**Issue by jimwhite** [polygon-io/client-python#819]
**Is your feature request related to a problem? Please describe.**
I'm using the flatfiles to process data for the entire market.  I need to be able to distinguish between actual stocks and test stocks.

**Describe the solution you'd like**
The TickerDetailsV3 API includes `is_test` in the response and I'd like that bool field to be added to and copied to the TickerDetails model class.

**Describe alternatives you've considered**
Currently I filter based on whether there is a CIK assigned.  Also I could use the raw API call to get the `is_test` value and then make the deserialization call to get a TickerDetails.  Neither of those are ideal and while I'm now fully educated about test tickers many folks coming are going to be surprised because this information isn't in the Python API.

**Comment by justinpolygon** [comment]
Hey @jimwhite, I'm looking at this now. What asset class are you seeing test symbols in flat files? I'm guessing stocks but wanted to make sure. My suspicion is we probably need to filter them out, from flat files, and that'll solve your problem. I'm just checking with the product team.

Thanks for such a detailed report it really help tracking things down and understanding what you're seeing.

**Comment by jimwhite** [comment]
Yes, us_stocks trades and quotes.  I'm making my own aggs from them (WIP: https://github.com/fovi-llc/zipline-polygon-bundle/blob/trades/zipline_polygon_bundle/trades.py) and I agree that test tickers should be filtered from those.

I doubt removing test tickers from those (trades and quotes) files is a real help because I assume we still have to deal with test tickers in the WebSocket streams (where they do their intended job) and I plan to also use the flatfiles for testing and running sims for my (eventual) real-time system.

The change I'd like to see for flatfiles is to have splits, dividends, and details for all tickers each day.  Currently I rely on caching those API calls (https://github.com/fovi-llc/zipline-polygon-bundle/blob/main/zipline_polygon_bundle/adjustments.py).  I'm using the Python API which is why I want is_test included in the TickerDetails model but if you published per day JSON flatfiles for those I'd use that instead.

**Comment by justinpolygon** [comment]
Great, thanks. I'll pass this along.

**Comment by jimwhite** [comment]
Thanks for the details but I don't agree with the decision to omit test tickers from trades and quotes because I still have to deal with them on my broker integration (currently Alpaca but same elsewhere).  Again, test tickers serve a necessary purpose in real-time streaming systems, so filtering from aggs is one thing and streaming is totally different.  I'm also pretty much of the same view of that now on the trades and quotes encoding.  At first I liked that Polygon unifies the three sources into one encoding but when I integrate with a broker I have to use the source format anyhow so it isn't actually helpful.

Since not all interests/use cases can be satisfied with a single approach my preferred approach would be to offer the original raw tape flat files in CSV and for your processed data that matches your API to be JSONL (or both CSV and JSONL).  The raw files are needed for (some) folks who implement streaming trading systems and the JSONL format is better than CSV because we could then use the same deserializer that the API uses.
